portfolio/src/main/webapp/index.html

```diff
@@ -1,19 +1,106 @@
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+</head>
+
+<body id="content">
+    <div class="container">
+        <div class="intro-container">
+            <div class="intro-text">
+                <h1>
+                    <span id="greeting-container">Hello</span>
+                    <span class="emoji">👋</span><br />
+                    My name is Collin</h1>
+                <h2>I'm a sophomore at Mount St. Mary's University ⛰</h2>
+            </div>
+            <div class="profile-image-container">
+                <img class="profile-image" src="images/profile.png" />
+            </div>
+        </div>
+        <div class="quick-links">
+            <a href="//linkedin.com/in/collingrimm">linkedin</a>
+            <a href="mailto:collins.geek@gmail.com">email</a>
+            <a href="//github.com/collinsnji">github</a>
+            <a href="//twitter.com/@grimmcollin">twitter</a>
+            <a href="//medium.com/@collin.grimm">blog</a>
+        </div>
+
+        <!-- section for bio -->
+        <div class="bio">
+            <div class="bio-text">
+                <h3 class="section-heading">Summary</h3>
+                <p>
+                    Collins Nji, who goes by the online alias "Collin Grimm," is a sophomore currently
+                    pursuing a
+                    BSc in Computer Science with a double minor in Data Science and English Language at Mount St. Mary's
+                    University in Maryland. Last summer, Collin interned as a Data Research Analyst at the Georgetown
+                    University
+                    <a href="//cset.georgetown.edu/" target="_blank">Center for Security and Emerging Technology.</a>
+                </p>
+            </div>
+            <div class="bio-text">
+                <h3 class="section-heading">Background</h3>
+                <p>
+                    I was born and raised in <a href="https://en.wikipedia.org/wiki/Cameroon" target="_blank">Cameroon
+                        🇨🇲</a>&mdash; a country in Cenntral Africa. I moved to the USA to continue my college
+                    education. I have four sisters, and unfortunately no brother. Cameroon is a bilingual country and
+                    the two official languages are English and French. I grew up sepaking both, but I also learned
+                    Arabic and Pidgin English (creole based English spoken in West Africa and Hawaii). Hence, I speak
+                    four languages in total, and I am currently learning German!
+                </p>
+            </div>
+            <div class="bio-text">
+                <h3 class="section-heading">Hobbies</h3>
+                <p>
+                    In my free time, I like to paint and draw. My drawings mostly depict abstact art scenes. I love
```

Comment:
nit: spelling mistake -> "abstract"

Reply:
Corrected the typo. Thanks for the feedback